### PR TITLE
Suggestions list fixes

### PR DESCRIPTION
**Pull request by richardtagger**
* SuggestionListItem uses pseudo-class :hover to highlight rather than involving JavaScript
* SuggestionListItem highlights with bold rather than underline at the request of design
* SuggestionList is now allowed to flow off-screen, working around Callout's default clipping behavior.
* Disables animation

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

**Comment by richardtagger** [comment]
@adrianmg @swese44 @unindented

File: src/components/SuggestionsList/SuggestionsList.css [hunk 1]
```diff
@@ -1,6 +1,10 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 @import "src/css/variables/colors.css";
 
+.y-suggestions-list.ms-Callout {
+  animation-name: none;
+}
```
Comment (richardtagger) [inline src/components/SuggestionsList/SuggestionsList.css]: @adrianmg here's a chance to change the box-shadow/border-radious stuff. The default looks like it matches Zeplin but I think you wanted to change it?

current settings:-
```cs
border-radius: 1px;
box-shadow: 0 0 12px 0 rgba(0, 0, 0, 0.2);
```

Comment (adrianmg) [reply src/components/SuggestionsList/SuggestionsList.css]: The border-radius might be applied to the container but is probably not being displayed properly due to the border-radius value of the first and last row.

Attaching the image showing this…

image

Comment (adrianmg) [reply src/components/SuggestionsList/SuggestionsList.css]: Left is wrong, right is correct

image

Comment (richardtagger) [reply src/components/SuggestionsList/SuggestionsList.css]: ah ok, taking a look

**Pull request merged**